libnavigation-ui/src/test/java/com/mapbox/navigation/ui/instruction/GuidanceViewImageProviderTest.kt

```diff
@@ -70,18 +67,14 @@ class GuidanceViewImageProviderTest {
 
     @Test
     fun `when guidance view url is null`() {
-        val captureData = slot<String>()
-        val bannerInstructions: BannerInstructions = mockk()
-        val bannerView: BannerView = mockk()
-        val bannerComponentList =
-            mutableListOf<BannerComponents>(BannerComponentsFaker.bannerComponentsWithNullGuidanceUrl())
-        every { bannerInstructions.view() } returns bannerView
-        every { bannerView.components() } returns bannerComponentList
+        val messageSlot = slot<String>()
+
+        every { bannerView.components() } returns mutableListOf(bannerComponentsWithNullGuidanceUrl())
 
         guidanceViewImageProvider.renderGuidanceView(bannerInstructions, callback)
 
-        verify(exactly = 1) { callback.onFailure(capture(captureData)) }
-        assertEquals("Guidance View Image URL is null", captureData.captured)
+        verify(exactly = 1) { callback.onFailure(capture(messageSlot)) }
+        assertEquals(ERROR_VIEW_IMAGE_URL_NULL, messageSlot.captured)
     }
 
     @Test
```

Comment:
`= coroutineRule.runBlockingTest` 👇 is not necessary

Reply:
done